src/sentry/core/endpoints/organization_details.py

```
@@ -775,6 +863,16 @@ class OrganizationDetailsPutSerializer(serializers.Serializer):
         help_text="The role required to download debug information files, ProGuard mappings and source maps.",
         required=False,
     )
+    hasGranularReplayPermissions = serializers.BooleanField(
+        help_text="Specify `true` to enable granular replay permissions, allowing per-member access control for replay data.",
+        required=False,
+    )
+    replayAccessMembers = serializers.ListField(
```

If `replayAccessMembers` is sent in a request without `hasGranularReplayPermissions` will those changes be applied or silently dropped?

---

The API request should go through and apply the changes nonetheless (e.g. to apply a list of privileged users first, and then switch to enforcing access), even when the flag is not turned on.